.github/ISSUE_TEMPLATE/Standard-Bug-Issue-Template.md

```diff
@@ -4,67 +4,67 @@ about: Use this template for all bugs
 labels: Bug
 ---
 
-## Story
-
+# Story
```

Comment:
Because this is a now a level 1 heading, from "Steps to Recreate" to "Notes" are sub-sections of "Story". Did you intend that?

Reply:
yes